### DISCUSSION TITLE
Html chunks as page

### DISCUSSION THREAD

**Discussion by Shubhamsrmn** [vercel/next.js#55744] [Help]
Hi I am using next js 13 app deployed on amplify aws but it gives me html chunks code after some time please tell me any solution.![Screenshot_20230921_194728.JPG](https://github.com/vercel/next.js/assets/91474475/66a3060a-5087-492e-9a12-dfb7e5f08fde)

**Comment by icyJoseph** [comment]
I think there is something you have to with your CDN to avoid this.

Could you check if this is your issue? https://github.com/vercel/next.js/issues/48569

**Comment by Shubhamsrmn** [comment]
That my issue but I'm using amplify aws for deployment of nextjs app. I
don't find solution for this issue. please provide solution for this issue

**Reply by mohdzaidnagori** [reply]
I am facing same issue but not getting any solution in internet regarding this also when deploy in vercel it is perfect work not any problem but in AWS amplify show that chunks
